puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_21_stable.yaml

```diff
@@ -1,12 +1,12 @@
 - job:
```

Comment:
In develop we also test this one with 2.5.

Reply:
added.